src/main/scala/junctions/crossing.scala

```diff
@@ -91,3 +90,46 @@ object ClockToSignal {
     c2s.io.signal_out
   }
 }
+
+object LevelCrossing {
```

Comment:
Yeah, I wasn't really sure what to call this that would make sense. It is, in-fact a "synchronizer for level signals".